view/src/components/farms.js

```diff
@@ -358,6 +423,340 @@ class Farms extends Component {
+                    <Chip className={classes.chip} label={tag} size="small" />
+                  ))}
+                  <Box
+                    display="flex"
+                    flexDirection="row"
+                    flexWrap="wrap"
+                    p={0}
+                    m={0}
+                  >
+                    <Box p={3}>
+                      <Typography className={classes.pos} color="textSecondary">
+                        Details:
+                      </Typography>
+                      <Typography variant="body2" component="p">
+                        Location of Farm: {`${farm.location.substring(0, 30)}`}
+                        <br />
+                        {`${farm.location.substring(30, 78)}`}
+                      </Typography>
+                    </Box>
+                    <Box p={3}>
+                      <Typography className={classes.pos} color="textSecondary">
+                        Point of Contact:
+                      </Typography>
+                      <Typography variant="body2" component="p">
+                        Name: {farm.contactName}
+                        <br />
+                        Phone: {farm.contactPhone}
+                        <br />
+                        Email: {farm.contactEmail}
+                      </Typography>
+                    </Box>
+                    <Box p={3}>
+                      <Typography className={classes.pos} color="textSecondary">
+                        Logistics:
+                      </Typography>
+                      <Typography variant="body2" component="p">
+                        Have Transportation Means:{" "}
+                        {farm.transportation ? "yes" : "no"}
+                        <br />
+                        Loading Dock: {farm.loadingDock ? "yes" : "no"}
+                        <br />
+                        Forklift: {farm.forklift ? "yes" : "no"}
+                      </Typography>
+                    </Box>
+                  </Box>
+                </CardContent>
+                <CardActions>
+                  {/* <Button size="small" color="primary" onClick={() => this.handleViewOpen({ farm })}>
+                                                {" "}
+                                                View{" "}
+                                            </Button> */}
+                  <Button
+                    size="small"
+                    color="primary"
+                    onClick={() => this.handleEditClick({ farm })}
+                  >
+                    Edit
+                  </Button>
+                  <Button
+                    size="small"
+                    color="primary"
+                    onClick={() => this.handleDelete({ farm })}
+                  >
+                    Delete
+                  </Button>
+                </CardActions>
+              </Card>
+            </Grid>
+          ))}
+        </Grid>
+      </div>
+    );
+  };
```

Comment:
a little worried about this chunk of code in general because it doesn't seem to reflect some changes that have been made and merged into master (if this is merged, a few pieces will break so we need to make sure that the code is up to date)

Reply:
updated from master manually -- it matches updates now